packages/contracts/scripts/deploy/dollar/solidityScripting/01_Diamond.s.sol

```diff
@@ -272,266 +271,50 @@ contract DiamondScript is Constants {
         );
-        selectorsOfCreditNftManagerFacet.push(
-            creditNftManagerFacet.setCreditNftLength.selector
+        selectorsOfStakingFacet = getSelectorsFromAbi(
+            "/out/StakingFacet.sol/StakingFacet.json"
         );
-        selectorsOfCreditNftManagerFacet.push(
-            creditNftManagerFacet.exchangeDollarsForCreditNft.selector
-        );
-        selectorsOfCreditNftManagerFacet.push(
-            creditNftManagerFacet.exchangeDollarsForCredit.selector
-        );
-        selectorsOfCreditNftManagerFacet.push(
-            creditNftManagerFacet.getCreditNftReturnedForDollars.selector
-        );
-        selectorsOfCreditNftManagerFacet.push(
-            creditNftManagerFacet.getCreditReturnedForDollars.selector
-        );
-        selectorsOfCreditNftManagerFacet.push(
-            creditNftManagerFacet.onERC1155Received.selector
-        );
-        selectorsOfCreditNftManagerFacet.push(
-            creditNftManagerFacet.onERC1155BatchReceived.selector
-        );
-        selectorsOfCreditNftManagerFacet.push(
-            creditNftManagerFacet.burnExpiredCreditNftForGovernance.selector
-        );
-        selectorsOfCreditNftManagerFacet.push(
-            creditNftManagerFacet.burnCreditNftForCredit.selector
-        );
-        selectorsOfCreditNftManagerFacet.push(
-            creditNftManagerFacet.burnCreditTokensForDollars.selector
-        );
-        selectorsOfCreditNftManagerFacet.push(
-            creditNftManagerFacet.redeemCreditNft.selector
-        );
-        selectorsOfCreditNftManagerFacet.push(
-            creditNftManagerFacet.mintClaimableDollars.selector
-        );
-
-        // Credit NFT Redemption Calculator
-        selectorsOfCreditNftRedemptionCalculatorFacet.push(
-            creditNftRedemptionCalculatorFacet.getCreditNftAmount.selector
-        );
-
-        // Credit Redemption Calculator
-        selectorsOfCreditRedemptionCalculatorFacet.push(
-            (creditRedemptionCalculatorFacet.setConstant.selector)
-        );
-        selectorsOfCreditRedemptionCalculatorFacet.push(
-            (creditRedemptionCalculatorFacet.getConstant.selector)
-        );
-        selectorsOfCreditRedemptionCalculatorFacet.push(
-            (creditRedemptionCalculatorFacet.getCreditAmount.selector)
-        );
-
-        // Dollar Mint Calculator
-        selectorsOfDollarMintCalculatorFacet.push(
-            (dollarMintCalculatorFacet.getDollarsToMint.selector)
+        selectorsOfStakingFormulasFacet = getSelectorsFromAbi(
+            "/out/StakingFormulasFacet.sol/StakingFormulasFacet.json"
         );
-        // Dollar Mint Excess
-        selectorsOfDollarMintExcessFacet.push(
-            (dollarMintExcessFacet.distributeDollars.selector)
+        selectorsOfTWAPOracleDollar3poolFacet = getSelectorsFromAbi(
+            "/out/TWAPOracleDollar3poolFacet.sol/TWAPOracleDollar3poolFacet.json"
         );
 
         // Credit Clock Facet
```

Comment:
@rndquu looks like you're leaving the recent facet selectors

Reply:
fixed